boot/zephyr/include/mcuboot_config/mcuboot_config.h

```diff
@@ -205,6 +205,10 @@
 #define MCUBOOT_SERIAL_WAIT_FOR_DFU
 #endif
 
+#ifdef CONFIG_BOOT_SERIAL_IMG_GRP_HASH
+#define BOOT_SERIAL_IMG_GRP_HASH
```

Comment:
Use the same prefixes as the entire mcuboot_config.h

Reply:
Fixed